mlir/include/mlir/Dialect/Tosa/IR/TosaComplianceData.h.inc

```diff
@@ -232,16 +233,19 @@ profileComplianceMap = {
         {i32T, i32T, i16T, i16T},
         {i32T, i32T, i32T, i32T}}}}},
     {"tosa.const",
-     {{{Profile::pro_int}, {{boolT}, {i8T}, {i16T}, {i32T}}},
+     {{{Profile::pro_int, Profile::pro_fp},
+       {{boolT}, {i8T}, {i16T}, {i32T}},
+       anyOf},
       {{Profile::pro_fp}, {{fp16T}, {fp32T}}}}},
     {"tosa.identity",
-     {{{Profile::pro_int},
-       {{boolT, boolT}, {i8T, i8T}, {i16T, i16T}, {i32T, i32T}}},
+     {{{Profile::pro_int, Profile::pro_fp},
+       {{boolT, boolT}, {i8T, i8T}, {i16T, i16T}, {i32T, i32T}},
+       anyOf},
       {{Profile::pro_fp}, {{fp16T, fp16T}, {fp32T, fp32T}}}}},
-    {"tosa.dim",
-     {{{Profile::pro_int, Profile::pro_fp}, {{boolT}}},
-      {{Profile::pro_int}, {{i8T}, {i16T}, {i32T}}},
-      {{Profile::pro_fp}, {{fp16T}, {fp32T}}}}},
+    {"tosa.variable_write",
```

Comment:
Is `tosa.variable` missing from this list?

Reply:
Yes, thanks for catching this mistake. The compilance metadata exporter from the spec repo doesn't retrieve the profile requirement for this op properly.